lidarrmetadata/provider.py

```diff
@@ -80,6 +81,18 @@ def _get_rate_limiter(key=None):
             "Don't know how to instantiate {}. Defaulting to NullRateLimiter".format(limit_class))
         return limit.NullRateLimiter()
 
+def response_url(url: str) -> str:
+    """
+    Transforms a URL to a response URL, which can take into account things such as a hosted cache
+    for third-party services.
+    """
+    parsed = urlparse(url)
+    if parsed.netloc.endswith("theaudiodb.com"):
+        parsed = parsed._replace(
+            netloc=CONFIG.IMAGE_CACHE_HOST,
+            path=f"v1/tadb/{parsed.path}"
```

Comment:
Seems this is not working on testing, but testing the function local seem to produce a url with extra parameters. Additionally it's got two `//` after tadb

```
https://imagecache.lidarr.audio/v1/tadb//images/media/artist/banner/tvyypy1476729221.jpg
```

it should remove `images/media` to produce url below

```
https://imagecache.lidarr.audio/v1/tadb/artist/banner/tvyypy1476729221.jpg
```